index.md

```diff
@@ -178,6 +182,11 @@ Examples and explanations for how pointers in C work. [Examples and explanations
 [Course notes](./CS136/CS136_Final.pdf){:.a#CS136--Notes--Stella},
 from sessions 6 through 11, by Stella and Chloe.
 
+[More course notes](https://grave-space-21c.notion.site/CS-136-5d2c3a59c9104336ad9ab7de54f7b9ff?pvs=4) by Benny
+
+[Mock final](./CS136/Benny_CS136_Final_Review_2024.pdf){:.a#CS136--Notes--Benny}, and [mock final solutions](./CS136/Benny_CS136_Final_Review__Solution_2024.pdf){:.a#CS136--Notes--Benny}, by Tommy and Benny
```

Comment:
Move the LaTeX files to `./CS136/*.tex` (same name as the PDF) and then use `{:l.#course--title--author}` (with `.l` instead of `.a`) to put the little "[source]" links to the LaTeX.

Reply:
Got it!